CHANGELOG.md

```diff
@@ -12,6 +12,30 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ## 🛠 Maintenance
 ## 📚 Documentation -->
 
+
+# [v0.1.0-alpha.1] 2021-11-18
+
+## :rocket::waxing_crescent_moon: Public alpha release
+
+> An alpha or beta release is in volatile, active development. The release might not be feature-complete, and breaking API changes are possible between individual versions.
+
+## :sparkles: Features
+
+- JSON log output
+
+## :hammer_and_wrench: Maintenance
+
+- Reduced CI build times
+- Improved overall performance by reducing the number of clones when composing and building a graphql response
+- Logging feature flags: The two remaining features available are "otlp-grpc" (on by default) and "otpl-http"
+- Move composition benchmarks to a separate crate
+- OTLP / Tracing updates
```

Comment:
I think It's mostly internal, since it doesn't have an impact on the release binary. I'm leaving it, and we will add an entry once we build with --all-features.